source/_docs/drush.md

```diff
@@ -365,10 +365,30 @@ See the [Drush Migrate documentation](https://drupal.org/node/1561820) for detai
 ## Known Limitations
 - Crontab: Currently, there is no way to manage Crontab on Pantheon. If you need a way to set up your own Cron interval, you can use an external cron service such as [Easy Cron](https://www.easycron.com/user/register).
 - The following Drush commands are not supported and will not work on Pantheon sites: `sql-sync-pipe`, `sql-sqlc`, `php-eval`. An alternative to `drush sql-sync` is to use `drush sql-dump` instead.
-- Incorrect `['uri']` in `pantheon.aliases.drushrc.php` file. Drush may fail if the `['uri']` array key has a different domain than what is expected by Drupal, resulting in the following error:
+- Drush may fail if the `['uri']` array key has a different domain than what is expected by Drupal, resulting in the following error:

```

Comment:
The failure here isn't so much that the `uri` is wrong -- usually, the "Drush command terminated abnormally" happens when folks add redirects that do not disable themselves when called from a cli tool.  The appropriate solution is to wrap any sort of drupal_goto(), or any other function that might call `exit()` in:

`if(php_sapi_name() != 'cli') { ... }`

There are some modules that require a correct URI in order to work correctly, though, so the basic advice in this document is still useful.